components/automate-gateway/handler/infra_proxy/orgs.go

```diff
@@ -53,13 +53,14 @@ func (a *InfraProxyServer) GetOrgByName(ctx context.Context, r *gwreq.GetOrgByNa
 	}, nil
 }
 
-// CreateOrg posts a org upstream
+// CreateOrg posts an org upstream
 func (a *InfraProxyServer) CreateOrg(ctx context.Context, r *gwreq.CreateOrg) (*gwres.CreateOrg, error) {
 	req := &infra_req.CreateOrg{
```

Comment:
Yes, At present for both servers and orgs, it is using system-generated GUIDs but while designing the schema I have already taken this into consideration so planing to use human-readable ids as:

 - For servers `name` will be used as ID and `description` would be `name` similar to teams services.

 - For orgs `name` will be used as ID. because it is unique and qualify for human-readable ID.

Will tackle this into further PR.

Reply:
Earlier, we had some discussion for the same and have provided the review comments https://github.com/chef/automate/pull/2209#discussion_r355164907